skycoin/skywire
# Fix/vpn server client close logs

Did you run `make format && make check`?
yes

Fixes #1301

 Changes:	
- Ignored errors if the read error is EOF or file already closed

How to test this PR:
1. Start vpn-server on visor A
2. From visor B connect to the vpn-server of visor A
3. Disconnect after a successful connection
4. Check server logs
